app/models/concerns/digital_object/datacite.rb

```diff
@@ -4,7 +4,7 @@ module DigitalObject::Datacite
   # or nil if minting did not go through
```

Comment:
I think this method description needs to be updated, since this method doesn't return nil anymore?

Reply:
Comment updated as suggested.